src/mobu/jupyterclient.py

```diff
@@ -501,10 +506,12 @@ def _remove_ansi_escapes(string: str) -> str:
         """
         return _ANSI_REGEX.sub("", string)
 
-    def _build_jupyter_spawn_form(self, image: JupyterImage) -> Dict[str, str]:
+    def _build_jupyter_spawn_form(
+        self, image: ControllerImage
+    ) -> Dict[str, List[str]]:
         """Construct the form to submit to the JupyterHub login page."""
         return {
-            "image_list": str(image),
-            "image_dropdown": "use_image_from_dropdown",
-            "size": self.config.image_size,
+            "image_list": [image.path],
+            "image_dropdown": [image.path],  # Not used
```

Comment:
Does this have to be sent? It would be nice if it could be omitted since it's not used.

Reply:
I think I can modify the models on the receiving end to make those optional.  At least one of list or dropdown needs to be sent, but I can certainly send dropdown as null.